### DISCUSSION TITLE
personas, real user experiences, etc. with low vision

### DISCUSSION THREAD

**Discussion by shawna-slh** [w3c/low-vision-a11y-tf#119] [General]
* How People with Disabilities Use the Web _(W3C WAI)_
  * **_{personas}_** Stories of Web Users:
    * [Lee, online shopper with color [vision deficiencies]](https://www.w3.org/WAI/people-use-web/user-stories/#shopper)
    * [Yun, retiree with low vision...](https://www.w3.org/WAI/people-use-web/user-stories/#retiree)
    * [Kaseem, teenager ... screen magnification software](https://www.w3.org/WAI/people-use-web/user-stories/#teenager)
   * **_{explanation}_** [Diverse Abilities and Barriers, Visual](https://www.w3.org/WAI/people-use-web/abilities-barriers/#visual)

* **_{real people}_** [User Experiences of people with low vision](http://www.tader.info/users.html) <- short summaries with some links to more details _(TAdER)_
  * [Angelita's User Experience: "Confessions of a Miserable Low Vision Computer User"](http://www.tader.info/users-angelita.html)
  * [Ben's User Experience "I'm just a geek in glasses."](http://www.tader.info/users-ben.html)
  * [Ramón's User Experience "I have retinitis pigmentosa"](http://www.tader.info/users-ramon.html)
  * [Sandra's User Experience "aging eyes"](http://www.tader.info/users-sandra.html)

* **_{persona}_** [Roger Retiree ... age-related macular degeneration... ](http://www.uiaccess.com/accessucd/personas_eg.html#retireesp) _(Just Ask: Integrating Accessibility Throughout Design)_

* **_{persona quotes}_** What's New in WCAG _(W3C WAI)_ 
  * in 2.1 for: [reflow](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#1410-reflow-aa), [non-text contrast](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#1411-non-text-contrast-aa), [text spacing](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#1412-text-spacing-aa), [content on hover](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#1413-content-on-hover-or-focus-aa)
  * in 2.2 for: [page break](https://www.w3.org/WAI/standards-guidelines/wcag/new-in-22/#2413-page-break-navigation-a)

* **_{explanation}_**  [Accessibility Requirements for People with Low Vision](https://www.w3.org/TR/low-vision-needs/#overview-of-low-vision) - Overview section includes explanations _(W3C WAI)_

**Comment by shawna-slh** [comment]
* **_{personas}_** [Maria Beuving - Aging](https://www.a11yportal.com/basics/essentials/personas.html#maria-beuving-aging),  [Emre Yavuz - Colour Blindness](https://www.a11yportal.com/basics/essentials/personas.html#emre-yavuz-colour-blindness) _(allyportal.com)_

* **_{persona}_** [Carles, a screen magnifier user](https://www.w3.org/WAI/RD/2013/e-learning/paper4/) _(e-learning symposium)_

* ? aegis-project.eu

**Reply by mraccess77** [reply]
We want to make sure to include a wide variety of people with visual impairments from minor to severe visual impairment with different careers/education and in different stages of life and diverse occupations as well as diverse cultures, name, various genders, and so forth.

Some examples include
* Contrast limitations due to cataracts
* Visual field loss due to retinitis pigmentosa
* Scotomas due to macular degeneration (AMD) or myopic degeneration
* Not being able to distinguish between colors because of color vision deficiency 
* Reduced acuity due to nystagmus or foveal hypoplasia (may be from Aniridia, Albinism, congenital visual impairment, etc.)
* Cortical/cerebral visual impairment (perhaps from birth or traumatic brain injury)

Possible groups
* Senior with AMD
* Person with albinism that has reduced acuity and that is glare sensitive due to nystagmus, foveal hypoplasia, and other factors
* Person with 20/70 visual acuity
* Person with 20/200 visual acuity

Different groups
* Students
* Seniors
* People in workforce with diverse fields and careers
*
